libs/experimental/langchain_experimental/graph_transformers/llm.py

```diff
@@ -56,10 +56,6 @@
                 "following input: {input}"
             ),
         ),
-        (
-            "human",
-            "Tip: Make sure to answer in the correct format and do not include any ",
```

Comment:
should we add this text to the preceding message?

Reply:
Sure, with OpenAI it doesn't make a difference, but it might with others.